devtools/install_python_requirements.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-cd ../app
+cd ./app
```

Comment:
Currently all of the devtools scripts are set with the expectation that they're run from the devtools directory. For consistency we should keep it the same. I'm not strongly opinionated whether everything should be from root or devtools. Thoughts @michielderoos ?

Reply:
My hunch is that if tools should be run from some directory, it should be their own. I've worked with tools before which store the app directory in an env variable though, which would make it not matter where it's run from. What if we have it read an env, and fall back to `.. /app` if it's not present?

Reply:
makes sense - shall i revert this change and change the README to reflect the preferred usage of running it from the directory?

Side note - I wonder if there's a pattern like npm's "script" manifest, or ruby's rake tool that people use for flask.  I ran into this same issue with another flask project I worked on

Reply:
yeah updating the readme would be perfect!

> Side note - I wonder if there's a pattern like npm's "script" manifest, or ruby's rake tool that people use for flask.

We use `flask_script Manager` for some things, which allows u to do thinkgs like
`manage.py db upgrade`, but so much of the scripting relies on non-python stuff to get things done.